.github/actions/vsce-publish/package.json

```diff
@@ -8,6 +8,6 @@
   "license": "SEE LICENSE IN ../../../LICENSE.txt",
   "type": "module",
   "dependencies": {
-    "@vscode/vsce": "2.21.1"
+    "@vscode/vsce": "^3.3.2"
```

Comment:
I hope this won't break the release 😅 I see that the "breaking" change from v3 is that they moved to Node 20 (which we are already using for GH actions).

Reply:
We will have to rip the bandaid at some point 😬